Обратная сторона реальности

yesiammark opened the pull request:
---
:mortar_board: [Обратная сторона реальности](https://up.htmlacademy.ru/react/13/user/2336671/tasks/6)

yesiammark committed:
Добавляет моки

yesiammark committed:
Подключает моки

yesiammark committed:
Правит тип оффера

yesiammark committed:
Убирает неиспользуемые компоненты

yesiammark committed:
Добавляет ссылку на офер и правит баги

yesiammark committed:
Добавляет отзывы

yesiammark committed:
Добавляет отображение комментариев

yesiammark committed:
Правит баги

HeleneGritsuk commented on src/components/place-card/place-card.tsx:
```diff
@@ -1,21 +1,41 @@
+      onMouseEnter={handleMouseEnter}
+      onMouseLeave={handleMouseLeave}
+    >
+      {offer.isPremium ? <PlaceCardMark /> : ''}
```
`{ offer.isPremium && <PlaceCardMark /> }`

yesiammark committed:
Добаляет сохранение стейтов

yesiammark committed:
Правит баги

yesiammark committed:
Правит className

yesiammark committed:
Добавляет стейт для сортировки

yesiammark committed:
Добавляет отображение сортировки

yesiammark committed:
Вносит различные исправления:
- Добавляет компонент список городов
- Добавляет информацию из моков на страницу офера
- Добавляет HelmetProvider
- Добавляет ScrollToTop на страницу офера

yesiammark committed:
Вносит правки:
- Случайные места не обновляются при ховере
- исправлены теги a на Link
- сохраняет id города на который наведена мышка
- выносит типы моков в отдельные файлы

yesiammark committed:
Правит баги
